app/modules/ads/bill-the-lizard-wrapper.js

```diff
@@ -103,7 +103,10 @@ export const billTheLizardWrapper = {
       context.set('services.billTheLizard.projects', config.projects);
       context.set('services.billTheLizard.timeout', config.timeout || 0);
 
-      billTheLizard.projectsHandler.enable('cheshirecat');
+      if (adsContext.opts.enableCheshireCat === true) {
```

Comment:
What do you think about storing this flag in context-service (`options.billTheLizard.cheshireCat`) in `setup.js` instead of passing here mediaWikiAdsContext object?

Reply:
done